generators/app/prompts.js

```diff
@@ -9,6 +13,21 @@ module.exports = {
           name: 'seedbank',
           checked: true,
           value: "gem 'seedbank'"
+        },
+        {
+          name: 'rack-cors',
+          checked: true,
+          value: "gem 'rack-cors'"
+        },
+        {
+          name: 'redis',
+          checked: true,
+          value: "gem 'redis', '~> 3.0'"
+        },
+        {
+          name: 'bcrypt',
+          checked: true,
+          value: "gem 'bcrypt', '~> 3.1.7'"
```

Comment:
I recently tried to use minitest in a project and I still think RSpec is important. FactoryGirl even more, fixtures are very limited. But I think minitest is great for small projects, and because of that, and it's support out-of-the-box, I think it should be the default.

I don't think minitest will really be enough in our projects, though.

I understand what you are saying and I think the whole community is moving in agreement with what you said. Including RSpec itself, which nowadays do a lot less invisible magic and comes with disabled monkey patching by default.